content/en/docs/contribute/style/style-guide.md

```diff
@@ -191,37 +194,46 @@ Set the value of `image` to nginx:1.16. | Set the value of `image` to `nginx:1.1
 Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
 {{< /table >}}
 
+However, consider quoting values where there is a risk that readers might confuse the value
+with an API kind.
+
 ## Referring to Kubernetes API resources
 
 This section talks about how we reference API resources in the documentation.
 
 ### Clarification about "resource"
 
 Kubernetes uses the word "resource" to refer to API resources, such as `pod`,
-`deployment`, and so on. We also use "resource" to talk about CPU and memory
+`deployment`, and so on. Documentation also uses "resource" to talk about CPU and memory
 requests and limits. Always refer to API resources as "API resources" to avoid
-confusion with CPU and memory resources.
+confusion with CPU and memory resources, or with other kinds of resource.
 
 ### When to use Kubernetes API terminologies
 
 The different Kubernetes API terminologies are:
 
-- Resource type: the name used in the API URL (such as `pods`, `namespaces`)
-- Resource: a single instance of a resource type (such as `pod`, `secret`)
-- Object: a resource that serves as a "record of intent". An object is a desired
+- _API kinds_: the name used in the API URL (such as `pods`, `namespaces`).
+  API kinds are sometimes also called _resource types_.
+- _API resource_: a single instance of an API kind (such as `pod`, `secret`).
+- _Object_: a resource that serves as a "record of intent". An object is a desired
   state for a specific part of your cluster, which the Kubernetes control plane tries to maintain.
+  All objects in the Kubernetes API are also resources.
+
+For clarity, you can add "resource" or "object" when referring to an API resource in docs.
+As an example, write "a Secret object" over just "a Secret".
```

Comment:
nit: Missing `the` in the phrase, `when referring to an API resource in docs.` 
nit: remove `over just` and  possibly change to `instead of`

Reply:
How's this now?